### PR TITLE
Remove webpack5 workaround

### PR DESCRIPTION
**Pull request by ger-benjamin**
[Examples](https://camptocamp.github.io/ngeo/refs/pull/9507/merge/examples/)
[Storybook](https://camptocamp.github.io/ngeo/refs/pull/9507/merge/storybook/)
[API help](https://camptocamp.github.io/ngeo/refs/pull/9507/merge/api/apihelp/apihelp.html)
[API documentation](https://camptocamp.github.io/ngeo/refs/pull/9507/merge/apidoc/)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "distlib": "tsc --pretty",
     "distlib-images": "copyfiles --verbose src/**/*.svg src/*.svg src/**/*.png src/*.png distlib",
     "distlib-css": "copyfiles --verbose src/**/*.css src/*.css src/**/*.scss src/*.scss distlib",
-    "distlib-css-bundle": "cp dist/desktop-*.css distlib/src/controllers/desktop.css && cp dist/mobile-*.css distlib/src/controllers/mobile.css && cp dist/iframe_api-*.css distlib/src/controllers/iframe_api.css",
+    "distlib-css-bundle": "cp dist/desktop-*.css distlib/src/controllers/ && cp dist/mobile-*.css distlib/src/controllers/ && cp dist/iframe_api-*.css distlib/src/controllers/",
```
[the remark this answers is not in the document]
Comment (ger-benjamin) [reply package.json:21]: Yes, ok, I forgot why you did that.
But `cp dist/desktop-*.css distlib/src/controllers/desktop.css` doesn't work => `destination is not a folder`

[the remark this answers is not in the document]
Comment (ger-benjamin) [reply package.json:21]: Wrong PR

**Review by sbrunner — APPROVED** [review]
For the second commit

**Comment by ger-benjamin** [comment]
I removed the 1st useless commit.
Now, it's possible that we have more to do in this PR... let's see what the CI say.

**Comment by sbrunner** [comment]
Looks to be a wrong reabase!

**Comment by ger-benjamin** [comment]
Yes, let me one second...

**Review by sbrunner — APPROVED** [review]
Nice :-)

**Pull request merged**